app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -137,6 +157,31 @@ class SignUpActivity : BaseActivity() {
         return isValid
     }
 
+    private fun validateName(string: String): Boolean {
```

Comment:
could you put this validation functions in another class, I think this could also be used for editing profile right? So 2 places to use, then its better to have it out of the SignUp activity. What do you think @Kavya-24 ?